roles/oneagent/tests/conmponent/scripts/tests/test_installAndConfig.py

```diff
@@ -0,0 +1,160 @@
+import logging
+from pathlib import Path, PureWindowsPath
+from typing import Set
+
+from command.platform_command_wrapper import PlatformCommandWrapper
+from util.common_utils import get_oneagentctl_path, get_platform_argument
+from util.constants.common_constants import HOST_SERVER_ADDRESS, HOST_SERVER_TOKEN
+from util.test_data_types import DeploymentPlatform, DeploymentResult
+from util.test_helpers import (
+    check_agent_state,
+    check_download_directory,
+    set_installer_download_params,
+    perform_operation_on_platforms,
+    run_deployment,
+)
+
+UNIX_DOWNLOAD_PATH = Path("/tmp/dyna")
+WINDOWS_DOWNLOAD_PATH = PureWindowsPath("C:\\tmp\\dyna")
+
+TECH_NAME_KEY = "orchestration_tech"
+TECH_VERSION_KEY = "tech_version"
+TECH_SCRIPT_VERSION_KEY = "script_version"
+
+INSTALLER_TAG = "install_tag"
+INSTALLER_PROPERTY = "install_prop1=install_1"
+PLATFORM_TAG = "platform_install_tag"
+PLATFORM_PROPERTY = "platform_install_property"
+CONFIG_INSTALL_TAG = "config_install_tag"
+CONFIG_INSTALL_PROPERTY = "install_prop2=config_install_prop"
+CONFIG_INTENDED_TAG = "config_intended_tag"
+CONFIG_INTENDED_PROPERTY = "install_prop2=config_intended_prop"
+
+CTL_TAGS_GETTER = "--get-host-tags"
+CTL_PROPERTIES_GETTER = "--get-host-properties"
+
+INSTALLER_ARGS = [f'--set-host-tag="{INSTALLER_TAG}"', f'--set-host-property="{INSTALLER_PROPERTY}"']
+INSTALLER_PLATFORM_ARGS = [f'--set-host-tag="{PLATFORM_TAG}"', f'--set-host-property="{PLATFORM_PROPERTY}"']
+CONFIG_INSTALL_ARGS = [f'--set-host-tag="{CONFIG_INSTALL_TAG}"', f'--set-host-property="{CONFIG_INSTALL_PROPERTY}"']
+CONFIG_INTENDED_ARGS = [f'--set-host-tag="{CONFIG_INTENDED_TAG}"', f'--set-host-property="{CONFIG_INTENDED_PROPERTY}"']
```

Comment:
name `CONFIG_INSTALL_ARGS` and `CONFIG_INTENDED_ARGS` with their purpose as currently, as we talked, it's unclear